openbas-api/src/main/java/io/openbas/utils/challenge/ChallengeExpectationUtils.java

```diff
@@ -0,0 +1,17 @@
+package io.openbas.utils.challenge;
+
+import io.openbas.rest.exercise.form.ExpectationUpdateInput;
+
+public class ChallengeExpectationUtils {
+
+  private ChallengeExpectationUtils() {}
+
+  public static ExpectationUpdateInput buildChallengeUpdateInput(Double score) {
+    ExpectationUpdateInput expectationUpdateInput = new ExpectationUpdateInput();
+    expectationUpdateInput.setSourceId("challenge");
```

Comment:
Maybe use a constant for all three of these strings? Also, how do we manage the translation on the frontside of this default name?